the-collab-lab/tcl-28-smart-shopping-list
# Isa - Single Item Styling

## Description

Added styling to the SingleItem component so that it more closely aligns with our wireframe.

- Referenced @ksiman14's code for the radio button styling and Material UI docs to create styling
- Imported `makeStyles` from MUI specifically for SingleItem styling
- Created styling for the div container, label, and delete button
- Put the styling inside a separate file called `SingleItemStyles.js` in src/themes
- Imported the styling from that file into the SingleItem component
- Imported `InputLabel` and `Button` components from MUI to replace existing label and delete button in SingleItem component
- Changed condition for "not soon" items from `> 30 && < 40` to `>= 30 && < 40` because "not soon" items were previously showing up as inactive

## Related Issue

Closes #50 

## Acceptance Criteria

Modify Single Item styling so that they reflect the what's in the wireframe, and match the soon, kind of soon, and not soon colors used in AddItem

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/56979810/131601385-fbea490f-f4c0-4095-8e08-9cb2f6745a25.png)

### After

![image](https://user-images.githubusercontent.com/56979810/131601273-7bb6600b-0836-4b5b-8b4a-8340b315dd06.png)
![image](https://user-images.githubusercontent.com/56979810/131753260-ff633fbc-8fbf-4c54-8e7b-64452d6ba1fc.png)

## Testing Steps / QA Criteria

- checkout the project url _or_ `git checkout ia-single-item`, then `npm start`
- see how you like the styling

NOTE: The styling for inactive elements probably won't show up because that still needs to be sorted out, but otherwise the styling is fine
Also note: When you hover over the delete button, it changes color as intended, but doesn't fill the entire silhouette for some reason. I'm not a fan, but I don't know how to fix it. For now, maybe it's okay?
